docs/processes/maintainers.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Maintainers playbook
+description: Process
+---
+
+<!--  markdownlint-disable MD043 -->
+
+## Overview
+
+!!! note "Please treat this content as a living document."
+
+This is document explains who the maintainers are, their responsibilities, and how they should be doing it. If you're interested in contributing,
+ see [CONTRIBUTING](https://github.com/aws-powertools/powertools-lambda-java/blob/main/CONTRIBUTING.md){target="_blank"}.
+
+## Current Maintainers
+
+| Maintainer            | GitHub ID                                                                       | Affiliation |
+|-----------------------|---------------------------------------------------------------------------------| ----------- |
+| Jerome Van Der Linden | [sthulb](https://github.com/jeromevdl){target="_blank"}          | Amazon      |
```

Comment:
```suggestion
| Jerome Van Der Linden | [jeromevdl](https://github.com/jeromevdl){target="_blank"}          | Amazon      |
```

Reply:
should we put AWS, rather than Amazon?

Reply:
[Python says Amazon](https://docs.powertools.aws.dev/lambda/python/latest/maintainers/), and I figure we're one company !